### DISCUSSION TITLE
Splunk distribution of OpenTelemetry for Kubernetes vs the original one

### DISCUSSION THREAD

**Discussion by donixa** [signalfx/splunk-otel-collector-chart#559] [Q&A]
What is the difference between Splunk distribution of OpenTelemetry for Kubernetes and the original one ?

**Comment by atoulme** [comment]
Which original distribution do you refer to? It could be that you are referring to https://github.com/splunk/splunk-connect-for-kubernetes or https://github.com/open-telemetry/opentelemetry-helm-charts ?

**Comment by donixa** [comment]
The later one, https://github.com/open-telemetry/opentelemetry-helm-charts

But if you can explain all of them, it would be great too 🙏

Thank you,

**Comment by govardha** [comment]
Hi @donixa 

I am definitely new to the splunk otel universe and k8s, but over the past few months I have learnt a LOT about how the splunk flavor of this helm chart is built and they have done a really good job in giving you a solid framework to develop/extend in your environment.

They give a good amount of examples which I used to build out everything.  

I would encourage you to read the documentation in the OTEL site and then Splunk's flavor of it and examples and stick with it, it eventually comes together :).
